core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerClientProvider.scala

```diff
@@ -28,6 +30,11 @@ trait BillingProfileManagerClientProvider {
 class HttpBillingProfileManagerClientProvider(baseBpmUrl: Option[String]) extends BillingProfileManagerClientProvider {
   def getApiClient(ctx: RawlsRequestContext): ApiClient = {
     val client: ApiClient = new ApiClient()
+
+    val clientConfig = new ClientConfig()
```

Comment:
See https://github.com/DataBiosphere/jade-data-repo/pull/1288/files#diff-608bbfbb0303e87fa7cb15128f663bd479532f68c996e27d031c68f241d628aeR90 and https://broadinstitute.slack.com/archives/C03B4RKTA5B/p1651527855852349 for more details. 

TL;DR: There's a limitation in the `HttpUrlConnection` library that the jersey `HttpUrlConnector` tries to workaround. The workaround doesn't work in Java 17 which prevents us from calling PATCH endpoints using our codegen clients in Rawls. Phil and Shelby were able to get it working using this fix which sets up a different connector provider. I have not stepped through this connector provider as closely, so I'm not sure exactly how it gets around the limitation, but I can confirm that it works in practice.